### ISSUE TITLE
The mcMMO packages aren't listed as a dependency in pom.xml

### ISSUE THREAD

**Issue by thehale** [toomanybugs1/Quick-Farm#3]
It was harder than expected to find these packages. If you can list them as dependencies in the pom.xml it would be much easier to help add new features to and do testing for this great plugin.

**Comment by toomanybugs1** [comment]
Unfortunately I don't think I can add mcMMO as a hard dependency as it's a paid plugin now. Though the free mcmmo classic might have the same api so I could look into that. What did you use to build?

**Comment by thehale** [comment]
To be honest, I didn't actually build it last night; I was just extremely careful to not remove any of your code until I was sure that my replacement implemented the same functionality. It was a bit tedious 😄 .

That said, this morning I did some research and found mcMMO's maven repository. I've gone ahead and added it as a Maven dependency in the pom.xml (see pull request #4 ). I've now successfully built the plugin and tested it on my server.

Just be aware that mcMMO has a GPL3.0 license. I believe that means you will have to put the same license on this plugin as long as you have a dependency on mcMMO.
